### PR TITLE
#2707 Update ketcher toolbar UI styles

### PR DESCRIPTION
**Pull request by Iogsotot**
Closes https://github.com/epam/ketcher/issues/2707
- [x] updated toolbar ui styles
- [x] added new component - `HorizontalDivider` (ketcher\packages\ketcher-react\src\script\ui\views\toolbars\TopToolbar\Divider.tsx)

image

### CODE REVIEW

File: packages/ketcher-react/src/components/styles/consts.ts [hunk 1]
```diff
@@ -18,7 +18,9 @@ const color = {
   white: '#FFFFFF',
   green: '#167782',
   lightGreen: '#188794',
-  graphite: '#333333'
+  graphite: '#333333',
+  'grey-4': '#cad3dd',
```
Comment (Nitvex) [inline packages/ketcher-react/src/components/styles/consts.ts:22]: I propose to call it like `gray` and `darkGray`

File: packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/AtomsList/AtomsList.tsx [hunk 2]
```diff
@@ -44,21 +43,21 @@ const StyledAtom = styled(Atom)((props: any) => {
   const atomColor = props?.el?.label ? ElementColor[props.el.label] : '#000'
   return `
        color: ${atomColor};
-       border: 1px solid ${atomColor};
+       border: 1px solid transparent;
        background-color: transparent;
        cursor: pointer;
        &:hover {
-         background-color: ${α(atomColor, 0.2)};
+         background-color: #EFF2F5;
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/ui/views/toolbars/RightToolbar/AtomsList/AtomsList.tsx:50]: Can we add a variable for this color?

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.module.less [hunk 3]
```diff
@@ -20,21 +20,19 @@
...
-    margin-bottom: 0;
     border-bottom: none;
-    padding-bottom: 0;
   }
-
 }
 
 .buttons {
-  overflow: hidden;
+  overflow: visible;
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.module.less:35]: Could you clarify, why `overflow` is required?

**Review by Nitvex — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.module.less (hunk 3 above)
Comment (Iogsotot) [reply packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.module.less:35]: toolbars shadow clipped if overflow: hidden

**Pull request closed without merging**